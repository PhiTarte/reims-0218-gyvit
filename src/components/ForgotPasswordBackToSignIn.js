import React from "react"

const ForgotPasswordBackToSignIn = () => <p>Back to Sign in</p>

export default ForgotPasswordBackToSignIn
