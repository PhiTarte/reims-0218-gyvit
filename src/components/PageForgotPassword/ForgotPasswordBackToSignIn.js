import React from "react"

const ForgotPasswordBackToSignIn = () => (
  <p style={{ marginTop: "10%" }}>Back to Sign in</p>
)

export default ForgotPasswordBackToSignIn
