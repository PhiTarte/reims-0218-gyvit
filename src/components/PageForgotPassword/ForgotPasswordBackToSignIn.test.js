//TODO CREATE TEST

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import ForgotPasswordBackToSignIn from "./ForgotPasswordBackToSignIn"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ForgotPasswordBackToSignIn />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<ForgotPasswordBackToSignIn />).toJSON()
  expect(tree).toMatchSnapshot()
})
