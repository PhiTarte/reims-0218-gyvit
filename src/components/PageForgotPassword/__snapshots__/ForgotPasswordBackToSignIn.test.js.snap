// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<p
  style={
    Object {
      "marginTop": "10%",
    }
  }
>
  Back to Sign in
</p>
`;
